handler.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Hindyadav/ked/pullRequest?prid=7538437)
```suggestion
"use strict";
```